components/script/dom/window.rs

```diff
@@ -1968,17 +1968,18 @@ impl Window {
                         new_url);
                     event.upcast::<Event>().fire(this.upcast::<EventTarget>());
                 });
-                // FIXME(nox): Why are errors silenced here?
-                let _ = self.script_chan.send(CommonScriptMsg::Task(
-                    ScriptThreadEventCategory::DomEvent,
-                    Box::new(
-                        self.task_manager
-                            .task_canceller(TaskSourceName::DOMManipulation)
-                            .wrap_task(task),
-                    ),
-                    self.pipeline_id(),
-                    TaskSourceName::DOMManipulation,
-                ));
+                self.script_chan
```

Comment:
This can actually be replaced by the syntax using `dom_manipulation_task_source`

Reply:
This comment is still relevant